debian/rules

```diff
@@ -1,114 +1,25 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
-# Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -Wno-format -Wno-format-security -g -DTAP_AND_HOLD
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
```

Comment:
These CFLAGS are silently disappeared in your commit.
It means the behavour of dpkg-buildpackage is changed.

Reply:
It has, yes. It had changed in many packages ported to dh10 before.
We don't use this in Jenkins, and nobody had requested we keep it in the rules file so far.

Reply:
It's a separate action which should be committed separately.
It's not connected to "porting to debhelper 10".

Reply:
Fine. I reverted it in 780bdf22fd425ba5acfd175baf70609250d7907c
For what I care it can just stay and we keep backwards compatibility.